src/Activities/Option/ActivityOption.cs

```diff
@@ -1,16 +1,21 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
-namespace Microsoft.Omex.Extensions.Abstractions.Option
+namespace Microsoft.Omex.Extensions.Activities.Option
 {
 	/// <summary>
 	/// Monitoring option
 	/// </summary>
-	public class MonitoringOption
+	public class ActivityOption
 	{
 		/// <summary>
 		/// Path to the setting
 		/// </summary>
 		public static string MonitoringPath = "Monitoring";
```

Comment:
Is this used anywhere?

Reply:
Yes 
<img width="369" alt="image" src="https://github.com/microsoft/Omex/assets/17307105/97c4d5be-d9a7-4a19-8a25-3abe34851634">